src/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+  margin-right: 4px;
+}
+
```

Comment:
The last star shouldn't have the right margin according to requirements ( last child)